prod_website/index.html

```diff
@@ -23,15 +23,30 @@
     />
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
     <script src="./js/preloader.js"></script>
+    <script>
+      function collapseprompt() {
+        var x = document.getElementById("prompt");
+        var image = document.getElementById("arrow");
+        if (x.style.display === "none") {
+          x.style.display = "block";
+          image.src = 'icons8-collapse-arrow-100.png';
+        }
+        else{
+          x.style.display = "none";
+          image.src = 'icons8-expand-arrow-100.png';
+        }
+      }
+    </script>
+
   </head>
 
   <body>
     <div class="se-pre-con"></div>
-    <header id="home">
-      <nav style="background-color: #fff; padding: 10px; display: flex; justify-content: space-between; align-items: center; width: 100%; box-sizing: border-box;">
+    <header id="home" style="margin-bottom: 0px;">
+      <nav style="background-color: #B3CEE5; padding: 10px; display: flex; justify-content: space-between; align-items: center; width: 100%; box-sizing: border-box;">
```

Comment:
It looks good, but I think we should make the background black so it's seamless with the rest of the header. Thoughts @katzyx @floriafz23 ?

Reply:
i agree